ignite --watch fails when Chrome isn't running

```
> ignite --watch

(node:3664) DeprecationWarning: Tapable.apply is deprecated. Call apply on the plugin directly instead
child_process.js:614
    throw err;
    ^

Error: Command failed: ps cax | grep "Google Chrome"
    at checkExecSyncError (child_process.js:574:11)
    at execSync (child_process.js:611:13)
    at Function.listening (/Users/adierkens/Developer/player-rewrite/player/node_modules/ignite/dist/ignite.js:256:41)
    at Nanobus._emit (/Users/adierkens/Developer/player-rewrite/player/node_modules/nanobus/index.js:160:14)
    at Nanobus.emit (/Users/adierkens/Developer/player-rewrite/player/node_modules/nanobus/index.js:26:10)
    at Server.server.once (/Users/adierkens/Developer/player-rewrite/player/node_modules/webpack-serve/lib/server.js:37:11)
    at Object.onceWrapper (events.js:255:19)
    at Server.emit (events.js:165:20)
    at emitListeningNT (net.js:1394:10)
    at process._tickCallback (internal/process/next_tick.js:152:19)
npm ERR! code ELIFECYCLE
```

--- a/package.json
+++ b/package.json
@@ -103,6 +103,7 @@
     "mini-css-extract-plugin": "^0.4.2",
     "mkdirp": "0.5.1",
     "nightmare": "3.0.1",
+    "opn": "^5.3.0",
     "postcss-loader": "^3.0.0",
     "postcss-nested": "^4.0.0",
     "postcss-simple-vars": "^5.0.1",

--- a/src/ignite.js
+++ b/src/ignite.js
@@ -3,6 +3,7 @@
 import fs from 'fs';
 import { execSync } from 'child_process';
 import path from 'path';
+import opn from 'opn';
 
 import history from 'connect-history-api-fallback';
 import convert from 'koa-connect';
@@ -255,16 +256,16 @@ export default async function build(options) {
         on: {
           listening: () => {
             if (finalOptions.open) {
-              execSync('ps cax | grep "Google Chrome"');
-              execSync(
-                `osascript ../src/chrome.applescript "${encodeURI(
-                  `http://localhost:${finalOptions.port}`
-                )}"`,
-                {
+              const url = encodeURI(`http://localhost:${finalOptions.port}`);
+              try {
+                execSync('ps cax | grep "Google Chrome"');
+                execSync(`osascript ../src/chrome.applescript "${url}"`, {
                   cwd: __dirname,
                   stdio: 'ignore'
-                }
-              );
+                });
+              } catch (error) {
+                opn(url);
+              }
             }
           }
         }

